selfdrive/car/hyundai/values.py

```diff
@@ -39,6 +36,9 @@ class Buttons:
   CAR.KIA_STINGER: [{
     67: 8, 127: 8, 304: 8, 320: 8, 339: 8, 356: 4, 358: 6, 359: 8, 544: 8, 576: 8, 593: 8, 608: 8, 688: 5, 809: 8, 832: 8, 854: 7, 870: 7, 871: 8, 872: 8, 897: 8, 902: 8, 909: 8, 916: 8, 1040: 8, 1042: 8, 1056: 8, 1057: 8, 1064: 8, 1078: 4, 1107: 5, 1136: 8, 1151: 6, 1168: 7, 1170: 8, 1173: 8, 1184: 8, 1265: 4, 1280: 1, 1281: 4, 1287: 4, 1290: 8, 1292: 8, 1294: 8, 1312: 8, 1322: 8, 1342: 6, 1345: 8, 1348: 8, 1363: 8, 1369: 8, 1378: 4, 1379: 8, 1384: 8, 1407: 8, 1419: 8, 1425: 2, 1427: 6, 1456: 4, 1470: 8
   }],
+  CAR.SANTA_FE: [{
```

Comment:
This one too?

Reply:
Alphabetical order. I think these changes are ok. Not sure about the dbc file change below though. Is the kia one a superset of the santa fe?

Reply:
There's no reason to add ambiguity into the naming of the dbc.
All Hyundai and Kia use the same DBC and it has absolutely nothing to do specifically with any model, and nothing at all to do specifically with santa Fe.
Easier to name things properly now.

Reply:
Historically it was all from the leaked dbc, I added a fix to an error in a message, and then you added a message regarding gear selection.
Both of these changes are true and valid for every vehicle encountered.

Reply:
ok. merged the opendbc repo. before I can merge this, you should update your openpilot repo with the latest dbc files.